spotbugs/etc/messages.xml

```diff
@@ -3712,17 +3725,16 @@ blocking and deadlock behavior. See <a href="http://www.javalobby.org/java/forum
     <Details>
       <![CDATA[
   <p> The code synchronizes on a boxed primitive constant, such as a Boolean.</p>
-<pre>
+<pre><code>
 private static Boolean inited = Boolean.FALSE;
-...
```

Comment:
I think this `...` is necessary, or code becomes non intuitive (why we can code fields and logic at the same place?)